grafana-tikv-dashboard.md

```diff
@@ -484,6 +484,44 @@ This section provides a detailed description of these key metrics on the **TiKV-
 - Initial Scanning Trigger Reason: The reason for triggering incremental scanning
 - Region Checkpoint Key Putting: The number of checkpoint operations logged to the PD
 
+### Import & Snapshot restore
```

Comment:
```suggestion
### Backup & Import
```
Please keep it consistent with the dashboard on Grafana
For v8.1.0 clusters, the dashboard is `Backup & Import`
<img width="297" alt="image" src="https://github.com/user-attachments/assets/90d309a7-db8e-4cdb-bddb-381165795d25">

Reply:
right, I guess it makes sense to change the dashboard first and later update the doc. I plan to rename the dashboard to be more explicit on restore vs import.

Reply:
well actually it already had a `backup` section in this doc that does not match any existing dashboard sections, I was adding the restore/import part of that.  let me know if think we still should add `backup` here